Update project environment, default config file, and AUTHORS

eauel opened the pull request:
-Update .gitignore to ignore test config file and IntelliJ IDEA project files
-Update AUTHORS file with new contributor
-Environment: Add perf and numcodecs libraries as dependencies
-Config file: Update file to include spaces for consistency
-Update benchmark config file to include new (future) options for ftp download and data conversion
-genomics-benchmark: create new entry point into benchmark tool when running from command-line

alimanfoo commented:
Hi @eauel, FWIW I would have thought you could just ignore the .idea directory, there are still some intellij files in the PR.

eauel commented:
Hi @alimanfoo, thanks for catching this. I have updated the .gitignore file to ignore the .idea directory, and I also removed all IDEA-related files from the project.
